plugins/inputs/nvidia_smi/schema_v12/parser.go

```diff
@@ -0,0 +1,75 @@
+package schema_v12
+
+import (
+	"encoding/xml"
+	"strconv"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs/nvidia_smi/common"
+)
+
+func Parse(acc telegraf.Accumulator, buf []byte) error {
+	var s smi
+	if err := xml.Unmarshal(buf, &s); err != nil {
+		return err
+	}
+
+	timestamp := time.Now()
+	if s.Timestamp != "" {
+		if t, err := time.Parse(time.ANSIC, s.Timestamp); err == nil {
+			timestamp = t
+		}
+	}
```

Comment:
See my comment on the bug report, but I think we may want to use time.Now() over what the timestamp says since we do not have timezone info?